azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/async_adapter.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from .mqtt_transport import MQTTTransport
+from azure.iot.common import asyncio_compat
+
+logger = logging.getLogger(__name__)
+
+
+class MQTTAsyncTransport(MQTTTransport):
```

Comment:
>MQTTAsyncTransport [](start = 6, length = 18)

For naming, we should be consistent about placement of "Sync" and "Async" modifiers.  It should always be a prefix or it should always be a suffix.  Putting it in the middle might flow off the tongue better, but the inconsistency is frustrating.

Reply:
I'd advocate for the end then, because putting Async at the beginning throws off alphabetization.

---
In reply to: [246941850](https://github.com/Azure/azure-iot-sdk-python-preview/pull/39#discussion_r246941850) [](ancestors = 246941850)